battleship-client/src/main/java/com/academy/solid/nie/client/ui/Board.java

```diff
@@ -218,18 +168,9 @@ private boolean isThereShipInTheNeighborhood(final int x, final int y) {
      * @param point - contains coordinates (x - horizontal, y - vertical)
      * @return true - if point is in the board scope
      */
-    private boolean isInScope(final Point2D point) {
-        return isInScope(point.getX(), point.getY());
-    }
-
-    /**
-     * Determines if the coordinates are in the range of the board.
-     *
-     * @param x - the horizontal coordinate of the cell
-     * @param y - the vertical coordinate of the cell
-     * @return
-     */
-    private boolean isInScope(final double x, final double y) {
+    private boolean isInScope(Point2D point) {
+        double x = point.getX();
+        double y = point.getY();
         return x >= 0 && x < MAX_HEIGHT && y >= 0 && y < MAX_WIDTH;
```

Comment:
This line is OK, but I just want to show that there is such thing as Range https://github.com/google/guava/wiki/RangesExplained